.github/workflows/test.yml

```diff
@@ -52,6 +52,7 @@ jobs:
             9.0.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-22.05 ;;
             9.2.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-22.05 ;;
             9.4.3) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-unstable ;;
```

Comment:
We can now reference tagged `nixpkgs/nixos-23.05` for GHC 9.4.5  (while you're here).  We could also get 9.6.1 from that same tag, although continuing to use nixos-unstable will get bugfix revisions in the 9.6 series... your call.

Reply:
I've converted both of these over to `nixos-23.05`. If we want to upgrade the 9.6 minor version in the future, we can always switch to using `nixos-unstable`.